tests/fast-integration/skr-test/.env.template

```diff
@@ -1,7 +0,0 @@
-KEB_HOST=
```

Comment:
Why did you remove it?

Reply:
Hm, I guess, just by mistake. Thanks, will revert it!